### PR TITLE
zulip-term: init at 0.5.2

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/zulip-term/pytest-executable-name.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/tests/cli/test_run.py b/tests/cli/test_run.py
+index 459aa82..c6e434e 100644
+--- a/tests/cli/test_run.py
++++ b/tests/cli/test_run.py
+@@ -180,7 +180,7 @@ def test_main_multiple_autohide_options(capsys, options):
+         assert str(e.value) == "2"
+     captured = capsys.readouterr()
+     lines = captured.err.strip('\n')
+-    lines = lines.split("pytest: ", 1)[1]
```
Comment (teto) [inline pkgs/applications/networking/instant-messengers/zulip-term/pytest-executable-name.patch:9]: any plan to upstream ?

**Review by teto — COMMENTED** [review]
passed nixpkgs-review. Could start the binary but didn't test login in a chatroom as my IDs got refused. (I use the connect by github usually, is that possible thorugh the client ?).

File: pkgs/applications/networking/instant-messengers/zulip-term/pytest-executable-name.patch (hunk 1 above)
Comment (dotlambda) [reply pkgs/applications/networking/instant-messengers/zulip-term/pytest-executable-name.patch:9]: I considered it, but I think the name is specific to our pytest infrastructure. I could split it using some regex.

**Comment by dotlambda** [comment]
> passed nixpkgs-review. Could start the binary but didn't test login in a chatroom as my IDs got refused. (I use the connect by github usually, is that possible thorugh the client ?).

See https://github.com/zulip/zulip-terminal#running-for-the-first-time. I just downloaded `zuliprc` from my account settings.

**Pull request merged**